### DISCUSSION TITLE
在Android11中，4.3.57版本还能通过gui/Surface来创建PAGSurface吗

### DISCUSSION THREAD

**Discussion by yaoziwei** [Tencent/libpag#2438] [Q&A] [answered]
我们需要在Android的native层（cameraservice）里面使用libpag，有gui/Surface（也就是nativewindow）对象，在4.3.57版本上还能通过MakeFrom来创建一个PAGSurface对象吗？

**Comment by kevingpqi123** [comment, marked as answer]
pag 的 API 保持前后版本兼容性，如果你们之前可以，后续的版本也可以
